src/main/java/com/telesign/util/TeleSignRequest.java

```diff
@@ -77,14 +86,25 @@
 	private AuthMethod auth = AuthMethod.SHA1;
 
 	/** Setting default Integer value that specifies the HttpConnection connect timeout value. Having default value as 30000 **/
-    private int connectTimeout = 30000;
+    private int connectTimeout;
 
     /** Setting default Integer value that specifies the HttpConnection read timeout value. Having default value as 30000 **/
-    private int readTimeout = 30000;
+    private int readTimeout;
     
     private String httpsProtocol = "TLSv1.2";
-
-
+    
+    /** User-Agent to track TeleSignJavaSDK and version being used **/
+    private final String USER_AGENT = "TeleSignJavaSDK/1.0"; 
+    
+    public static String RESOURCE_DIR = "src/test/resources/";
+    
+    private final String TEST_URL = "https://localhost:1443";

```

Comment:
Not crazy about having the testing so tightly integrated into the codebase. Please remove the testing service concept and we will just include samples for now.